tests/pdgid/test_functions.py

```diff
@@ -130,6 +130,22 @@ def test_is_quark(PDGIDs):
         assert is_quark(id) == False
 
 
+def test_is_sm_quark(PDGIDs):
+    _sm_quarks = (
+        PDGIDs.DQuark,
+        PDGIDs.UQuark,
+        PDGIDs.SQuark,
+        PDGIDs.CQuark,
+        PDGIDs.BQuark,
+        PDGIDs.TQuark,
+    )
+    _non_sm_quarks = [id for id in PDGIDs if id not in _sm_quarks]
+    for id in _sm_quarks:
+        assert is_sm_quark(id) == True
```

Comment:
```suggestion
        assert is_sm_quark(id)
```

Never use `==` with the literals `True` and `False`. Either use `is` if you really want to make sure that it returns a bool, or just check directly.

Reply:
(same below in other additions - I'm also fixing existing ones based on Flake8 reports in a branch)

Reply:
Good catch - this is a copy and paste from other similar code but it did not trigger my eye.